dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/transform/strategy/PrivatePropertyRemoveStrategy.java

```diff
@@ -0,0 +1,35 @@
+package com.dotmarketing.portlets.contentlet.transform.strategy;
+
+import static com.dotmarketing.portlets.contentlet.transform.strategy.TransformToolBox.privateInternalProperties;
+
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import java.util.Map;
+import java.util.Set;
+
+public class PrivatePropertyRemoveStrategy extends AbstractTransformStrategy<Contentlet> {
+
+    PrivatePropertyRemoveStrategy(final TransformToolBox toolBox) {
+        super(toolBox);
+    }
+
+    @Override
+    public Contentlet fromContentlet(final Contentlet contentlet) {
+        return contentlet;
+    }
+
+    @Override
+    public Map<String, Object> transform(final Contentlet contentlet, final Map<String, Object> map,
+            Set<TransformOptions> options) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'options' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5596402)